modules/rules_foreign_cc/0.11.1/presubmit.yml

```diff
@@ -0,0 +1,11 @@
+matrix:
+  platform: ["centos7", "debian10", "macos", "ubuntu2004", "windows"]
+  bazel: ["6.x", "7.x"]
+
+tasks:
+  verify_targets:
+    name: "Verify build targets"
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+      - "@rules_foreign_cc//toolchains:make_tool"
```

Comment:
Can you fix https://buildkite.com/bazel/bcr-presubmit/builds/6523#01907e18-502a-4b7f-b5aa-551181b0fcbd ?